cpp/src/gandiva/llvm_generator_test.cc

```diff
@@ -110,8 +110,10 @@ TEST_F(TestLLVMGenerator, TestAdd) {
   auto ir = generator->engine_->DumpIR();
   EXPECT_THAT(ir, testing::HasSubstr("vector.body"));
 
-  EvalFunc eval_func = (EvalFunc)generator->engine_->CompiledFunction(fn_name);
+  EXPECT_OK_AND_ASSIGN(auto fn_ptr, generator->engine_->CompiledFunction(fn_name));
+  EXPECT_NE(fn_ptr, nullptr);
```

Comment:
Should we use `ASSERT_*`?

Reply:
Updated. I don't realize their difference previously. Thanks.